docs/future/core/reference/corectl.md

```diff
@@ -228,6 +231,52 @@ Argument:
 
 * **address**: The listen address, in `host:port` format, to be added to the allowed members list.
 
+
+### `get`
+
+Retrieves the current value of a configuration option.
+
+```
+corectl get [key]
+```
+
+Argument:
+
+* **key**: The configuration option to retrieve.
+
+
+### `add`
+
+Adds a tuple to a configuration option's set of values. The configuration option must be defined as a set. The number of value arguments provided must match the configuration option's tuple length. Add will error if a conflicting tuple already exists in the set. To overwrite the existing tuple, provide the `-u` flag.
```

Comment:
> A configuration option is identified by a key, and consists of a set of tuples.

The tricky part to communicate is that an option consists _either_ of a set of tuples or a single tuple. Some options you can add multiple values to, others only one. For now we only have sets but we should probably be careful with our language so that we don't confuse. I wouldn't be surprised if scalar option values are introduced before 1.3 ships.